app/models.py

```diff
@@ -26,31 +30,36 @@ def delete(self):
     @classmethod
     def get_by_id(cls, _id):
         return cls.query.filter_by(id=_id).first()
-    
+
     @classmethod
     def get_all(cls):
         return cls.query.all()
 
+
 class User(UserMixin, BaseModel):
+    __schema__ = UserSchema()
+
     username = db.Column(db.String(64), index=True, unique=True)
     email = db.Column(db.String(120), index=True, unique=True)
     password_hash = db.Column(db.String(128))
-    posts = db.relationship('Post', backref='author', lazy='dynamic')
+    posts = db.relationship("Post", backref="author", lazy="dynamic")
 
     def __repr__(self):
-        return '<User {}>'.format(self.username)
-    
+        return "<User {}>".format(self.username)
+
     def set_password(self, password):
         self.password_hash = generate_password_hash(password)
 
     def check_password(self, password):
         return check_password_hash(self.password_hash, password)
 
-    
+
 class Post(BaseModel):
+    __schema__ = PostSchema()
```

Comment:
Same as above. This is wrong in the issue. The __schema__ attribute needs to have the Class not the object as value